src/jit/gentree.cpp

```diff
@@ -6308,6 +6308,10 @@ GenTreePtr          Compiler::gtCloneExpr(GenTree * tree,
         }
         copy->gtCall.gtRetClsHnd = tree->gtCall.gtRetClsHnd;
 
+#if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
```

Comment:
These kinds of bugs will get exposed if structDesc field is initialized with invalid values.  Looking at gtNewCallNode() i don't see structDesc is not initialized.